ui/src/components/Search/FilterChips.scss

```
@@ -32,8 +33,17 @@
   }
 
   &__add {
-    display: inline-flex;
+    display: flex;
     align-items: center;
-    gap: 0.35rem;
+    gap: $pad-small;
+    // Take a full row so the value control has room to grow; when chips are present this wraps onto its own line below them.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)